ziti-probe/subcmd/run.go

```diff
@@ -5,14 +5,6 @@ package subcmd
 import (
```

Comment:
Let's just delete ziti-probe for now. We can resurrect it from git history if we need.